torch/csrc/cudnn/Conv.cpp

```diff
@@ -225,6 +225,11 @@ void cudnn_convolution_forward(
   size_t workspaceSize;
   CHECK(cudnnGetConvolutionForwardWorkspaceSize(handle, idesc.desc, wdesc.desc,
       cdesc.desc, odesc.desc, fwdAlg, &workspaceSize));
+  if (workspaceSize > 1024*1024*1024) {
```

Comment:
One should not hardcode maximum workspace size in this way - many big-memory cards can and do use workspace bigger than 1 GB